app/utils.py

```diff
@@ -388,26 +388,87 @@ def get_host(host=None):
         return host
 
 
+# List of sensitive field names in person and note records.
+SENSITIVE_FIELDS = [
+  'date_of_birth',
+  'author_email',
+  'author_phone',
+  'email_of_found_person',
+  'phone_of_found_person',
+]
```

Comment:
Note that home_street is the street name, not street number. We explicitly asks users not to include number:
https://screenshot.googleplex.com/dz75VyeHqfv
So it may be considered non-sensitive?

Anyway I'm just preserving the existing list here. Modifying the set of sensitive fields should be in a separated pull request and requires product decision. Can you ask Pete if you really want to add it?

Reply:
Ah, ok, if it's an existing list then this is the wrong place to discuss it.